webserver.js

```diff
@@ -3225,12 +3225,18 @@ module.exports.CreateWebServer = function (parent, db, args, certificates, doneF
                 // Load default page style or new modern ui
                 var uiViewMode = 'default';
                 var webstateJSON = JSON.parse(webstate);
+                // Check in priority order: query param > domain config > saved user preference
                 if (req.query.sitestyle != null) {
                     if (req.query.sitestyle == 3) { uiViewMode = 'default3'; }
-                } else if (webstateJSON && webstateJSON.uiViewMode == 3) {
-                    uiViewMode = 'default3';
+                    else if (req.query.sitestyle == 78) { uiViewMode = 'default78'; }
```

Comment:
erm this will break your meshcentral web ui?
the `default78` is the file name and the is no `default78.handlebars` file!